src/gameobjects/rendertexture/RenderTexture.js

```diff
@@ -378,17 +386,23 @@ var RenderTexture = new Class({
             this.renderer.setFramebuffer(this.framebuffer);
 
             var gl = this.gl;
-    
+            
+            gl.enable(gl.SCISSOR_TEST);
```

Comment:
Don't do this unless the w/h !== the canvas w/h, it just adds un-needed gl ops.

Reply:
ok, I'll add the check for w/h. I've checked a few examples and SCISSOR_TEST was always enabled at the start of the canvas draws. I also didn't find any disable command in the phaser.js. Is it necessary to enable it? I mean is it possible that it is not enabled?

Reply:
Yes it needs enabling here.